### DISCUSSION TITLE
Add multiline review comment?

### DISCUSSION THREAD

**Discussion by alex-popov-tech** [pwntester/octo.nvim#301] [Q&A] [answered]
Hey, thanks for such a nice plugin!

Question - how can i add multiline comment in review? in didn't find any visual mapping, neither selecting and calling `Octo comment add` working for thin case :(

**Comment by pwntester** [comment, marked as answer]
Hi!

You have to use `:OctoAddReviewComment` or `:OctoAddReviewSuggestion` after selecting the visual selection. I think it should be mapped to `<space>ca` or `Octo comment add`. I had some problems implementing range command that way but I would revisit it and try to get it working as people expect. Do you mind opening an issue for it?

**Reply by alex-popov-tech** [reply]
done, here you go #303 
thank you for answer, that works!
another question tho -  i wasn't able to find commands you said in README or doc, which probably means it will be confusing people further until #303 ...does it makes sense to add some clarification in readme now? because it says:

```
Add comments with <space>ca or suggestions with <space>sa on single or multiple visual-selected lines
```

which says that it works on multi-line, but its not
